tests/run-spark-fvt-deploy.yaml

```diff
@@ -0,0 +1,34 @@
+#
+# (C) Copyright IBM Corp. 2021, 2021
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+apiVersion: v1
+kind: Pod
+metadata:
+  name: POD_NAME_PLACEHOLDER
+  namespace: CLUSTER_NAMESPACE_PLACEHOLDER
+  labels:
+    app: POD_NAME_PLACEHOLDER
+spec:
+  restartPolicy: Never
+  imagePullSecrets:
+    - name: all-icr-io
+  containers:
+  - name: spark-fvt-test-app
+    image: us.icr.io/vpc-dev-cohort-rns/cohort-spark-fvt:1.0.0
+    command: ["sleep"]
+    args: ["28800"]
```

Comment:
I'm confused by this sleep. What is it trying to accomplish?

Reply:
That was something done so as to to be able to exec into the pod even after the run is done.  Once a pod goes into Completed status you do not have an ability to exec into the pod. This provides a window in time to be able to do that. Not sure i would need to do it but kept it there as that is what i do also for the other pod is spin of for the taurus based tests.